app/lib/utils.ts

```diff
@@ -238,3 +238,7 @@ export function arrayIncludes (array: readonly (string | number)[], value: strin
 export function objectEntries <Obj> (obj: Obj) {
   return Object.entries(obj) as ObjectEntries<Obj>
 }
+
+export function moveArrayElement (array, oldIndex, newIndex) {
+  return array.splice(newIndex, 0, array.splice(oldIndex, 1)[0])
```

Comment:
```suggestion
  array.splice(newIndex, 0, array.splice(oldIndex, 1)[0])
  return array
```
Returning the `array.splice` returned value is less useful than returning the array itself, see use in component

Reply:
done in 0543120ca